firebase-perf/src/test/java/com/google/firebase/perf/metrics/AppStartTraceTest.java

```diff
@@ -261,4 +279,38 @@ public void testFirebasePerfProviderOnAttachInfo_initializesGaugeCollection() {
     Assert.assertEquals(oldSessionId, SessionManager.getInstance().perfSession().sessionId());
     verify(mockPerfSession, times(2)).isGaugeAndEventCollectionEnabled();
   }
+
+  @Test
+  @Config(sdk = 26)
+  public void timeToInitialDisplay_isLogged() {
```

Comment:
Can you add some comments to clarify what is going on in this test? There is a lot of setup and some verify statements mid way through the test making it hard to follow.

Reply:
Added comments! The setup is quite big, only the last few lines of this test is actually verifying.